tests/resources/bootstrap-jpm.js

```diff
@@ -0,0 +1,385 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is an unmodified file from the add-on SDK. It needs to be included as-is so the stock checksums will match it.